packages/manager/modules/octavia-load-balancer/src/octavia-load-balancer.module.js

```diff
@@ -3,16 +3,17 @@ import '@ovh-ux/manager-core';
 import '@ovh-ux/ng-ui-router-breadcrumb';
 import '@uirouter/angularjs';
 import 'angular-translate';
+import 'ovh-api-services';
```

Comment:
**nitpick:** `ovh-api-services` is deprecated. Can you remove it?

Reply:
Done